apps/book/schema.py

```diff
@@ -53,7 +54,9 @@ class Meta:
 
 
 class BookType(DjangoObjectType):
-    quantity_in_cart = graphene.Int(required=True)
+    cart_details = graphene.Field(CartItemType)
```

Comment:
Let's define BookDetails and define cart details there. That way we don't need to use data loader if it's hard to implement.

Reply:
@thenav56  I have added data loader